### PR TITLE
editShow: Replace checkboxes with toggle-button

### PR DESCRIPTION
**Pull request by p0psicles**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: themes-default/slim/views/editShow.mako [hunk 1]
```diff
@@ -50,7 +52,8 @@ const startVue = () => {
                     {text: 'Ignored', value: 'Ignored'}
                 ],
                 seriesLoaded: false,
-                saving: false
+                saving: false,
+                enableButton: true
```
Comment (sharkykh) [inline themes-default/slim/views/editShow.mako]: Unused?

File: themes-default/slim/views/editShow.mako [hunk 2]
```diff
@@ -225,14 +228,16 @@ const startVue = () => {
                         <div class="form-group">
                             <label for="subtitles" class="col-sm-2 control-label">Subtitles</label>
                             <div class="col-sm-10 content">
-                                <input type="checkbox" id="subtitles" name="subtitles" v-model="series.config.subtitlesEnabled"/> search for subtitles
+                                <toggle-button width="45" heigth="22" id="subtitles" name="subtitles" v-model="series.config.subtitlesEnabled" :sync="true"></toggle-button>
```
Comment (sharkykh) [inline themes-default/slim/views/editShow.mako]: 1. Boolean properties can be: `<toggle-button ... sync></toggle-button>`
3. There's a typo in the height property (not `heigth`)
2. I wonder if we can somehow "subclass" the component to set our preferred defaults, so we wouldn't have to keep adding `width="45" height="22"` everywhere.
@OmgImAlexis Mixins maybe? I'm not sure I understand the docs ATM.

Looks like these apply to all the buttons.

Comment (OmgImAlexis) [reply themes-default/slim/views/editShow.mako]: The width and height should be set in the component with props then just set the prop to have a default of these values.

Comment (OmgImAlexis) [reply themes-default/slim/views/editShow.mako]: Ah, I see it's a third-party component, we could add a mixin I think.

Comment (p0psicles) [reply themes-default/slim/views/editShow.mako]: ok will add the component

Comment (p0psicles) [reply themes-default/slim/views/editShow.mako]: Tried it but couldnt get it to work. I suggest keep it like is.

File: themes-default/slim/views/editShow.mako (hunk 1 above)
Comment (p0psicles) [reply themes-default/slim/views/editShow.mako]: Ill remove this and then merge

File: themes-default/slim/views/editShow.mako (hunk 2 above)
Comment (sharkykh) [reply themes-default/slim/views/editShow.mako]: @p0psicles I was able to figure it out :)

**Pull request merged**